### PR TITLE
[PAN-2752] Add findPrivacyGroup endpoint

### PR DESCRIPTION
**Pull request by Puneetha17**
`findPrivacyGroup` returns the list of all the details of the privacy groups the given addresses are a part of.

### CODE REVIEW

File: src/main/java/net/consensys/orion/enclave/QueryPrivacyGroupPayload.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  }
+
+  @JsonProperty("privacyGroupId")
+  public List<String> privacyGroupId() {
+    return privacyGroupId;
+  }
+
+  @JsonProperty("privacyGroupToAppend")
+  public String privacyGroupToAppend() {
+    return privacyGroupToAppend;
```
Comment (ekellstrand) [inline src/main/java/net/consensys/orion/enclave/QueryPrivacyGroupPayload.java]: What is privacyGroupToAppend?

Comment (Puneetha17) [reply src/main/java/net/consensys/orion/enclave/QueryPrivacyGroupPayload.java]: It is the "new" privacy group to be added to the existing list of privacy groups corresponding to the addresses.

File: src/main/java/net/consensys/orion/cmd/Orion.java [hunk 2]
```diff
@@ -496,8 +521,10 @@ private KeyValueStore createStorage(String storage, Path storagePath) {
       }
     } else if (storage.toLowerCase().startsWith("sql")) {
       try {
-        return SQLKeyValueStore.open(db);
-      } catch (IOException e) {
+        // FIXME: Hack to enable update in SQL.
+        // TODO: Update net.consensys.cava.kv.SQLKeyValueStore to enable updates
```
[the remark this answers is not in the document]
Comment (Puneetha17) [reply src/main/java/net/consensys/orion/cmd/Orion.java]: A JI is raised here https://pegasys1.atlassian.net/browse/PAN-2886

File: src/acceptance-test/java/net/consensys/orion/acceptance/EthClientStub.java [hunk 3]
```diff
@@ -119,6 +124,51 @@ ReceiveResponse receivePrivacy(String digest, String publicKey) {
     return payloadFuture.join();
   }
 
+  public Optional<PrivacyGroup> createPrivacyGroup(String[] addresses, String from, String name, String description) {
+    PrivacyGroupRequest createGroupRequest = new PrivacyGroupRequest(addresses, from, name, description);
+    CompletableFuture<PrivacyGroup> keyFuture = new CompletableFuture<>();
+    httpClient.post(clientPort, "localhost", "/privacyGroupId").handler(resp -> {
```
[the remark this answers is not in the document]
Comment (Puneetha17) [reply src/acceptance-test/java/net/consensys/orion/acceptance/EthClientStub.java]: Raised PR https://github.com/PegaSysEng/orion/pull/251

**Comment by Puneetha17** [comment]
Waiting for https://github.com/PegaSysEng/orion/pull/251 to be merged

**Pull request merged**